js/apps/admin-ui/src/PageHeader.tsx

```diff
@@ -172,13 +172,18 @@ export const Header = () => {
     );
   };
 
+  const logo = environment.logo ? environment.logo : "/logo.svg";
+  const logoUrl = environment.logoUrl
+    ? environment.logoUrl
+    : toDashboard({ realm });
+
   return (
     <PageHeader
       showNavToggle
       logo={
-        <Link to={toDashboard({ realm })}>
+        <Link to={logoUrl}>
```

Comment:
I am trying to think of a use-case for this one, under what circumstances would one want to replace the default URL here? Is this something we supported historically as well?

Reply:
One use case is where you have a suite of apps that all have the same look and feel.  Clicking on the logo should go the same place for all apps.

Reply:
Is anyone doing or requiring this right now? I feel like being able to change this at least for the Admin Console would be a bit of an anti-feature to be honest.

Reply:
> Is anyone doing or requiring this right now? I feel like being able to change this at least for the Admin Console would be a bit of an anti-feature to be honest.

I disagree.  It's very common for clicking on a logo to take you to the home page for whatever that logo represents.  That's what a user often expects.  So if someone wants that behavior we should let them do it.

Also, customization should be as similar as possible between admin and account console.